test/e2e/config.yaml

```diff
@@ -18,3 +18,44 @@ languages:
       - "helloworld.go"
       - "service.yaml"
       - "Dockerfile"
+  - language: "kotlin"
+    expectedOutput: "Hello Kotlin Sample v1"
+    copies:
+      - "/src/main/kotlin/com/example/hello/Main.kt"
+      - "build.gradle"
+      - "service.yaml"
+      - "Dockerfile"
+  - language: "nodejs"
+    expectedOutput: "Hello Node.js Sample v1!"
+    copies:
+      - "index.js"
+      - "package.json"
+      - "service.yaml"
+      - "Dockerfile"
+  - language: "php"
+    expectedOutput: "Hello PHP Sample v1!\\n"
```

Comment:
I suggest your comparison ignores trailing blanks, LF or CR.

Reply:
The output is actually not a LF, it's a string `\n`, that's why I used double `\` here.

Reply:
This doesn't seem right. Looks like an error in the sample. If so, please fix it.

Reply:
I do have a plan for consolidating all these outputs, will address this as well